libs/agno/agno/agent/agent.py

```diff
@@ -1667,9 +1738,7 @@ def continue_run(
         # Update session state from DB
         session_state = self._update_session_state(session=agent_session, session_state=session_state)
 
-        # Resolve dependencies
-        if self.dependencies is not None:
-            self._resolve_run_dependencies()
+        # Dependencies should already be resolved in main run() method
```

Comment:
hmm no  if you passed dependencies on `run`, then it won't be persisted (no state) so you'd have to pass it again on continue. 
If you passed it on the agent itself, then sure.